tests/TestCase/Cache/Engine/MemcachedEngineTest.php

```diff
@@ -786,10 +786,10 @@ public function testConfigurationConflict()
         $this->assertEquals('yay', Cache::read('duration_test', 'long_memcached'), 'Value was not read %s');
         $this->assertEquals('boo', Cache::read('short_duration_test', 'short_memcached'), 'Value was not read %s');
 
-        sleep(1);
+        usleep(500000);
```

Comment:
0.5 seconds? Then I think that not only the `long_memcached` cache, but also the `short_memcached` cache is available here. If the following assertion passed here, this test case wouldn't make sense.
```php
$this->assertEquals('boo', Cache::read('short_duration_test', 'short_memcached'), 'Value was not read %s');
```